### PR TITLE
Add 'Unstable' trigger to the other extendedEmail signature

### PR DESCRIPTION
**Pull request by marcesher**
I missed this when I submitted #58

### CODE REVIEW

**Comment by vickumar1981** [comment]
:+1:

**Comment by imuchnik** [comment]
@marcesher can you rebase please, and  I will merge.

**Comment by marcesher** [comment]
@imuchnik conflicts resolved. I also changed the implementation of one of the functions to simply call the other addExtendedEmail(), rather than the block of code.

File: src/main/groovy/jenkins/automation/utils/CommonUtils.groovy [hunk 1]
```diff
@@ -36,18 +36,8 @@ class CommonUtils {
...
-                triggers.each {
-                    trigger(triggerName: it,
-                            sendToDevelopers: sendToDevelopers, sendToRequester: sendToRequester, includeCulprits: includeCulprits, sendToRecipientList: sendToRecipientList)
-
-                }
-            }
-        }
-
+    static void addExtendedEmail(context, List<String> emails, List<String> triggers = ["Failure", "Unstable", "Fixed"], sendToDevelopers = false,  sendToRequester = true, includeCulprits = false, sendToRecipientList = true) {
+        addExtendedEmail(context, emails.join(","), triggers, sendToDevelopers, sendToRequester, includeCulprits, sendToRecipientList)
```
Comment (imuchnik) [inline src/main/groovy/jenkins/automation/utils/CommonUtils.groovy:40]: Stunning.

**Pull request merged**